### PR TITLE
Create operator 1.10.1 release branch

### PR DESCRIPTION
**Pull request by haibinxie**

### CODE REVIEW

File: deploy/operator.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ spec:
     spec:
       containers:
       - name: portworx-operator
-        image: portworx/px-operator:1.10.0-dev
+        image: portworx/px-operator:1.10.1-dev
         imagePullPolicy: Always
         command:
         - /operator
```
Comment (pure-jliao) [inline deploy/operator.yaml:26]: line 27, do we still need the `--verbose` here?

**Review by pure-jliao — APPROVED** [review]
one question regarding the verbose flag, otherwise LGTM

File: deploy/operator.yaml (hunk 1 above)
Comment (haibinxie) [reply deploy/operator.yaml:26]: any idea who uses this operator.yaml? the customer facing ones are spec gen and CSV which have been removed.

Comment (pure-jliao) [reply deploy/operator.yaml:26]: I'm not sure but I think we can merge the change first.

Comment (piyush-nimbalkar) [reply deploy/operator.yaml:26]: No one outside uses this. It is just for development.

**Pull request merged**